frontend/app/src/people/main/header.tsx

```diff
@@ -510,6 +510,7 @@ const MTab = styled.div<TagProps>`
 `;
 
 const LoggedInBtn = styled.div`
+  width: 130px;
```

Comment:
What would this look like if the user had a long username? Would we be able to instead limit to a max-width and give it an elipsiese if it gets too long

For example
usernamexyz12345 -> usernamex...

Reply:
@kevkevinpal  just checking this now and it works like that already, added screenshot to see if you are satisfied with the current way.

<img width="861" alt="Screenshot 2023-04-25 at 11 02 01" src="https://user-images.githubusercontent.com/44719946/234243720-acc47fe9-858e-4869-8323-9fe4db53141e.png">



<img width="861" alt="Screenshot 2023-04-25 at 11 04 24" src="https://user-images.githubusercontent.com/44719946/234244523-beb07ee7-81f2-4ea1-8558-63ed8908d72a.png">